src/interchange/src/avro/envelope_debezium/deduplication.rs

```diff
@@ -110,6 +111,31 @@ impl DebeziumDeduplicationStrategy {
             pad_start,
         })
     }
+
+    pub fn render<G: Scope<Timestamp = Timestamp>>(
+        self,
+        collection: Collection<G, Row, Diff>,
+        debug_name: String,
+        worker_index: usize,
+        arity: usize,
+        key_indices: Option<Vec<usize>>,
+    ) -> Collection<G, Row, Diff> {
+        let mut state = DebeziumDeduplicationState::new(self, key_indices);
+        collection.flat_map(move |row| {
+            let should_use = state
+                .as_mut()
+                .map(|d| {
+                    d.should_use_record(None, &debug_name, worker_index, &row, arity - 2, arity - 1)
```

Comment:
similar to above, I'm a bit uncomfortable with these magic numbers with no docs.